src/http/http_jwt.h

```diff
@@ -30,14 +30,14 @@ namespace http
 
   struct JwtPayload
   {
-    size_t nbf;
     size_t exp;
     std::string iss;
+    std::optional<size_t> nbf;
```

Comment:
Thank you for discovering and adding this! Still find it interesting how IdPs differ in various small things like this.